cdec/installation-manager-tests/src/main/resources/bin/install/test-install-update-im-cli-client.sh

```diff
@@ -23,17 +23,12 @@ printAndLog "TEST CASE: Install and update IM CLI client"
 
 vagrantUp ${SINGLE_NODE_VAGRANT_FILE}
 
-# install IM CLI 4.2.0 and restore default directory and $PATH settings in .bashrc
+# install IM CLI
 installImCliClient ${PREV_IM_CLI_CLIENT_VERSION} --install-directory=codenvy-im
-executeSshCommand "mv ~/codenvy-im/cli ~/codenvy-im/codenvy-cli"
-executeSshCommand "sed -i 's|export CODENVY_IM_BASE=/home/vagrant/.*||' ~/.bashrc &> /dev/null"
-executeSshCommand "sed -i 's|\$CODENVY_IM_BASE/cli/bin|/home/vagrant/codenvy-im/codenvy-cli/bin|' ~/.bashrc &> /dev/null"
```

Comment:
Because it was needed for migration from IM CLI 4.2.0 to 4.3.0, and isn't needed anymore.